src/modals/AddProjectModal.tsx

```diff
@@ -116,8 +111,9 @@ const AddProjectModal = () => {
                   boxShadow: "none",
                   outline: "none"
                 }}
-                value={name}
-                onChange={handleNameChange}
+                onChange={(event) => {
```

Comment:
We **dont** need **onChange** when we use useRef :D please accept this reality :D